dependency/actions/compile/jammy.Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:jammy
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get -y update && \
+  apt-get -y install openssl1.1 libffi-dev libssl-dev autoconf bison gperf ruby zlib1g-dev libyaml-dev curl build-essential
```

Comment:
Do we include `openssl1.1` because older versions of ruby require that version? And newer versions use openssl v3 (probably included by `libssl-dev`)?

Reply:
Ah!!!! Great catch. This should be openssl3 on the jammy side. I was testing some stuff out to make jammy compilation work for older versions of Ruby but it didn't work. I'll fix.

Reply:
You probably don't explicitly need to add `openssl3` because `libssl-dev` will likely pull it in as a required dependency.

But adding it explicitly is fine, too.